llvm/test/CodeGen/AIE/aie2/dyn-stackalloc.ll

```diff
@@ -53,27 +53,29 @@ define void @test_loop_dyn_alloca(i32 noundef %n) {
 ; CHECK-LABEL: test_loop_dyn_alloca:
 ; CHECK:         .p2align 4
 ; CHECK-NEXT:  // %bb.0: // %entry
-; CHECK-NEXT:    paddb [sp], #32; nopa ; nops ; nopxm ; nopv
```

Comment:
Somehow we used to simplify copies to/from `SP`, I'd rather not touch it for now.